packages/tables/resources/views/components/selection/indicator.blade.php

```diff
@@ -36,23 +36,29 @@ class="text-sm font-medium leading-6 text-gray-700 dark:text-gray-200"
     </div>
 
     <div class="flex gap-x-3">
-        <x-filament::link
-            color="primary"
-            :id="$this->getId() . '.table.selection.indicator.record-count.' . $allSelectableRecordsCount"
-            tag="button"
-            :x-on:click="$selectAllRecordsAction"
-            :x-show="$allSelectableRecordsCount . ' !== ' . $selectedRecordsPropertyName . '.length'"
-        >
-            {{ trans_choice('filament-tables::table.selection_indicator.actions.select_all.label', $allSelectableRecordsCount) }}
-        </x-filament::link>
+        {{ \Filament\Support\Facades\FilamentView::renderHook(\Filament\Tables\View\TablesRenderHook::SELECTION_INDICATOR_ACTIONS_BEFORE, scopes: static::class) }}
 
-        <x-filament::link
-            color="danger"
-            tag="button"
-            :x-on:click="$deselectAllRecordsAction"
-        >
-            {{ __('filament-tables::table.selection_indicator.actions.deselect_all.label') }}
-        </x-filament::link>
+        <div class="flex gap-x-3">
```

Comment:
Do we really need another wrapping div here, @danharrin?

Reply:
The render hook name wouldnt make sense if it wasnt directly wrapping an element IMO, could be confusing, BEFORE and AFTER hooks always directly surround the same element

Reply:
Is it clear from the PR description if `before` and `after` should be used or `start` and `end`?